<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color : #FFF;
	margin : 0px;
	padding : 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="content">
     <property name="styleSheet">
      <string notr="true">QFrame#content {
	background-color : transparent;
	border: 1px solid #DADDE1;
	border-radius: 5px;
	min-width : 380px;
	min-height : 285px;
}

QLabel {
	font : 77 12px &quot;Arial&quot;;
	color : green;
}
#fme_input_user, #fme_input_password {
	background-color : transparent;
	border-radius : 5px;
	border : 1px solid #DADDE1;
	min-height : 40px;
}

QLineEdit {
	background-color : transparent;
	border : none;
	color : #616161;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton#btn_login {
	background-color : #1877F2;
}

QPushButton#btn_login:pressed {
	background-color : transparent;
	color : #1877F2;
	border: 1px solid #1877F2;
	border-radius: 5px;
}

QPushButton#btn_signup {
	background-color : #42B72A;
}

QPushButton#btn_signup:hover, #btn_exit:hover, #btn_login:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton#btn_signup:pressed {
	background-color : transparent;
	color : #42B72A;
	border: 1px solid #42B72A;
	border-radius: 5px;
}

QPushButton#btn_exit {
	background-color : #616161;
}

QPushButton#btn_exit:pressed {
	background-color : transparent;
	color : #616161;
	border: 1px solid #616161;
	border-radius: 5px;
}

QPushButton#btn_forgotten_password {
	border-style: outset;
	border-width: 0px;
	color : #1877F2;
	font : 77 15px &quot;Arial&quot;;
}

QPushButton#btn_forgotten_password:hover {
	text-decoration: underline;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string> * Ingrese sus datos de inicio de sesión.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="fme_input_user">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLineEdit" name="tfd_user">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string> Ingrese su usuario</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fme_input_password">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLineEdit" name="tfd_password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string> Ingrese su contraseña</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_forgotten_password">
        <property name="text">
         <string>¿Olvidaste tu contraseña?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_login">
        <property name="text">
         <string>Iniciar sesión</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_signup">
        <property name="text">
         <string>Crear cuenta</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
